profiles/w3c.js

```diff
@@ -59,6 +59,7 @@ const modules = [
   import("../src/core/anchor-expander.js"),
   import("../src/core/custom-elements/index.js"),
   import("../src/core/web-monetization.js"),
+  import("../src/core/beforeSave.js"),
```

Comment:
Might want to add it to other profiles too as it's a core feature.

Reply:
I'll let there other folks do that, as it's up to them to maintain their own profiles.